assets/datatype.css

```
@@ -0,0 +1,49 @@
+var {
+    position: relative;
+    cursor: pointer;
+    display: inline-block;
+  }
+  
+  var[data-type] {
+    --tooltip-background: #000;
+    --tooltip-color: #fff;
+  }
+  
+  var[data-type]::before,
+  var[data-type]::after {
+    position: absolute;
+    left: 50%;
+    top: -6px;
+    opacity: 0;
+    transition: opacity 0.4s;
+  }
+  
+  /* the triangle or arrow or caret or whatever */
+  var[data-type]::before {
+    content: "";
+    transform: translateX(-50%);
+    border-width: 4px 6px 0 6px;
+    border-style: solid;
+    border-color: transparent;
+    border-top-color: var(--tooltip-background);
+  }
+  /* actual text */
+  var[data-type]::after {
+    content: attr(data-type);
+    transform: translateX(-50%) translateY(-100%);
+    background: var(--tooltip-background);
+    text-align: center;
+    /* additional styling */
+    color: var(--tooltip-color);
+    font-family: "Dank Mono", "Fira Code", monospace;
+    font-style: normal;
+    padding: 6px;
+    border-radius: 3px;
+    color: rgb(218,202,136);
```

For consistency how we're using colors everywhere else

```suggestion
    color: #daca88;
```

---

Will make the change.